### ISSUE TITLE
Install instruction is wrong on Windows

### ISSUE THREAD

**Issue by saibing** [NvChad/nvchad.github.io#144]
![image](https://user-images.githubusercontent.com/5676390/228788254-376f0bc3-a2de-43b1-b569-77a442412a5e.png)

Windows not support env linux style variable $HOME.  the install instrucion only work in git bash, not work in powershell.

**Comment by siduck** [comment]
I don't use windows,  which would work on windows?

**Comment by Neeraj029** [comment]
> Windows not support env linux style variable $HOME

![image](https://user-images.githubusercontent.com/66945520/229028734-3a742ba9-97c9-4213-8136-3e62969b36fd.png)
works here 🤔

**Comment by lucario387** [comment]
depending on the version of powershell it will work

if using cmd.exe then $HOME doesn't work

**Comment by lucario387** [comment]
tl;dr: note if using `cmd.exe`, or if user using `pwsh.exe`

**Comment by Neeraj029** [comment]
![image](https://user-images.githubusercontent.com/66945520/229029163-1d42d676-5e3d-4ee1-9f34-aba6f04efd63.png)
Btw is && supported on PS? or Do I have outdated version of it @lucario387

**Comment by lucario387** [comment]
i don't know and i wont try to know as my windows drive is still borked as of now

**Comment by Neeraj029** [comment]
Nvm on the latest version of PS, the command works perfectly fine

**Comment by MasterMach50** [comment]
The command only works on powershell, not on cmd.
This should be specified in the install page

These paths will work on cmd
```
%LOCALAPPDATA%\nvim\
%USERPROFILE%\AppData\Local\nvim
C:\Users\%USERNAME%\AppData\Local\nvim
```

**Comment by siduck** [comment]
> The command only works on powershell, not on cmd. This should be specified in the install page
> 
> These paths will work on cmd
> 
> ```
> %LOCALAPPDATA%\nvim\
> %USERPROFILE%\AppData\Local\nvim
> C:\Users\%USERNAME%\AppData\Local\nvim
> ```

so this right?

```
git clone https://github.com/NvChad/NvChad %LOCALAPPDATA%\nvim\ --depth 1 && nvim
```

**Comment by siduck** [comment]
> But why don't just mention both? Or mention that this is to install through cmd.exe

yes, just confirming before I make the change to the docs

**Comment by TheBearodactyl** [comment]
> ![image](https://user-images.githubusercontent.com/5676390/228788254-376f0bc3-a2de-43b1-b569-77a442412a5e.png)
> 
> Windows not support env linux style variable $HOME. the install instrucion only work in git bash, not work in powershell.

it works just fine w/ PWSH7
